src/Symfony/Component/Security/Http/Firewall/RememberMeListener.php

```diff
@@ -38,14 +39,15 @@ class RememberMeListener implements ListenerInterface
     /**
      * Constructor.
      *
-     * @param TokenStorageInterface          $tokenStorage
-     * @param RememberMeServicesInterface    $rememberMeServices
-     * @param AuthenticationManagerInterface $authenticationManager
-     * @param LoggerInterface                $logger
-     * @param EventDispatcherInterface       $dispatcher
-     * @param bool                           $catchExceptions
+     * @param TokenStorageInterface                  $tokenStorage
+     * @param RememberMeServicesInterface            $rememberMeServices
+     * @param AuthenticationManagerInterface         $authenticationManager
+     * @param LoggerInterface                        $logger
+     * @param EventDispatcherInterface               $dispatcher
+     * @param bool                                   $catchExceptions
+     * @param SessionAuthenticationStrategyInterface $sessionStrategy
      */
-    public function __construct(TokenStorageInterface $tokenStorage, RememberMeServicesInterface $rememberMeServices, AuthenticationManagerInterface $authenticationManager, LoggerInterface $logger = null, EventDispatcherInterface $dispatcher = null, $catchExceptions = true)
+    public function __construct(TokenStorageInterface $tokenStorage, RememberMeServicesInterface $rememberMeServices, AuthenticationManagerInterface $authenticationManager, LoggerInterface $logger = null, EventDispatcherInterface $dispatcher = null, $catchExceptions = true, SessionAuthenticationStrategyInterface $sessionStrategy)
```

Comment:
The new argument must be optional (people can use it outside the framework which would break their applications when upgrading).